app/src/components/submissionPage/GetCharacterCountRemaining.test.tsx

```diff
@@ -0,0 +1,24 @@
+import MentionElements from "../../types/MentionElements";
+import GetExtraLength from "./GetExtraLength";
+import { GetCharacterCountRemaining } from "./CharactersLeftDisplay";
+import Snap from "../../firebase/snap/Snap";
+import ValidateSnap from "./ValidateSnap";
+
+describe("<SubmissionTextBox />", () => {
```

Comment:
This test has nothing to do with `<SubmissionTextBox />`. It should be renamed.

Reply:
This function actually lives in the file CharactersLeftDisplay, so I've moved its tests to that test file.